bootstrap/aws/util.sh

```diff
@@ -44,7 +20,7 @@ apollo_launch() {
 }
 
 ansible_ssh_config() {
-  pushd $APOLLO_ROOT/terraform/aws
+  pushd "${APOLLO_ROOT}/terraform/${APOLLO_PROVIDER}"
     BASTION_IP=$(terraform output bastion.ip)
```

Comment:
we ```export APOLLO_bastion_ip``` earlier, then we get from terraform again, could we just use APOLLO_bastion_ip?